src/analyzer.js

```diff
@@ -92,21 +92,52 @@ function getViewerData(bundleStats, bundleDir, opts) {
     const assetBundles = statAsset.isChild ? getChildAssetBundles(bundleStats, statAsset.name) : bundleStats;
     const modules = assetBundles ? getBundleModules(assetBundles) : [];
     const asset = result[statAsset.name] = _.pick(statAsset, 'size');
+    const assetSources = bundlesSources && _.has(bundlesSources, statAsset.name) ?
+      bundlesSources[statAsset.name] : null;
 
-    if (bundlesSources && _.has(bundlesSources, statAsset.name)) {
-      asset.parsedSize = Buffer.byteLength(bundlesSources[statAsset.name]);
-      asset.gzipSize = gzipSize.sync(bundlesSources[statAsset.name]);
+    if (assetSources) {
+      asset.parsedSize = Buffer.byteLength(assetSources.src);
+      asset.gzipSize = gzipSize.sync(assetSources.src);
     }
 
     // Picking modules from current bundle script
-    asset.modules = _(modules)
-      .filter(statModule => assetHasModule(statAsset, statModule))
-      .each(statModule => {
-        if (parsedModules) {
+    const assetModules = modules.filter(statModule => assetHasModule(statAsset, statModule));
+
+    // Adding parsed sources
+    if (parsedModules) {
+      const unparsedEntryModules = [];
+
+      for (const statModule of assetModules) {
+        if (parsedModules[statModule.id]) {
           statModule.parsedSrc = parsedModules[statModule.id];
+        } else if (isEntryModule(statModule)) {
+          unparsedEntryModules.push(statModule);
         }
-      });
+      }
+
+      // Webpack 5 changed bundle format and now entry modules are concatenated and located at the end on the it.
```

Comment:
Is here a typo? The "at the end on the it" confused me and I'm not sure what it refers to, so I'm afraid I'm not able to provide a code suggestion to fix it.

Reply:
Hah, yep, it should be `and located at the end of it.`